browser/main/SideNav/index.js

```diff
@@ -65,37 +144,17 @@ class SideNav extends React.Component {
         style={style}
       >
         <div styleName='top'>
+          <div styleName='switch-buttons'>
+            <button styleName={location.pathname.match(/tag/) ? 'non-active-button' : 'active-button'} onClick={(e) => this.handleSwitchFoldersButtonClick(e)}>Folders</button>
+            <button styleName={location.pathname.match(/tag/) ? 'active-button' : 'non-active-button'} onClick={(e) => this.handleSwitchTagsButtonClick(e)}>Tags</button>
```

Comment:
writing same logic twice does non make sense...
and it it hard to understand.
How about declare variable like `isTagActive`?

```javascript
const isTagActive = location.pathname.match(/tag/)
```